app/views/fe/questions/fe/_date_field.html.erb

```diff
@@ -1,6 +1,6 @@
 <% locked = date_field.locked?(params, @answer_sheet, @presenter, current_person) %>
 <%= calendar_date_select_tag "answers[#{date_field.id}]", date_field.format_date_response(@answer_sheet),
-      class: "#{"form-control" if Fe.bootstrap} text " + date_field.validation_class(@answer_sheet, @page),
+      class: "#{"form-control" if Fe.bootstrap} text " + date_field.validation_class(@answer_sheet, @page) + date_field.css_class,
```

Comment:
Sorry, this should be `&.css_class` in case an enclosing app is nil.

Also, it seems weird to me that there's no forced space. If there are styles in `date_field.validation_class(@answer_sheet, @page) ` it will append it directly without a space, which doesn't seem right.

`      class: "#{"form-control" if Fe.bootstrap} text #{date_field&.css_class} " + date_field.validation_class(@answer_sheet, @page)`

maybe? There will be an extra space after text if css_class isn't set, but that's fine for the css field.